recipe/gen_patch_json.py

```diff
@@ -568,17 +568,16 @@ def _gen_new_index(repodata, subdir):
             #         depends.append("blas 1.* openblas")
             #         instructions["packages"][fn]["depends"] = depends
 
-        if 'snappy >=1.1.7,<1.1.8.0a0' in deps:
-            i = record['depends'].index('snappy >=1.1.7,<1.1.8.0a0')
-            record['depends'][i] = 'snappy >=1.1.7,<2.0.0.0a0'
+        _replace_pin('zstd >=1.4.0,<1.4.1.0a0', 'zstd >=1.4.0,<1.5.0.0a0', deps, record)
+        _replace_pin('zstd >=1.4.1,<1.4.2.0a0', 'zstd >=1.4.1,<1.5.0.0a0', deps, record)
+        _replace_pin('zstd >=1.4.2,<1.4.3.0a0', 'zstd >=1.4.2,<1.5.0.0a0', deps, record)
+        _replace_pin('zstd >=1.4.3,<1.4.4.0a0', 'zstd >=1.4.3,<1.5.0.0a0', deps, record)
+        _replace_pin('zstd >=1.4.4,<1.4.5.0a0', 'zstd >=1.4.4,<1.5.0.0a0', deps, record)
+        _replace_pin('zstd >=1.4.5,<1.4.6.0a0', 'zstd >=1.4.5,<1.5.0.0a0', deps, record)
```

Comment:
Can we use loosen_pin helper function

Reply:
@isuruf To what are you referring? I don't see such a function in the repo.